packages/cli/__tests__/zosfiles/__system__/delete/vsam/cli.files.delete.vsam.system.test.ts

```diff
@@ -120,14 +120,26 @@ describe("Delete VSAM Data Set", () => {
         });
 
         it("should delete a VSAM data set that has a retention period", async () => {
-            let response = runCliScript(__dirname + "/__scripts__/command/command_invoke_ams_define_for_statement.sh",
-                TEST_ENVIRONMENT, [dsname, volume]);
-            response = runCliScript(__dirname + "/__scripts__/command/command_delete_vsam_data_set.sh",
+            const response = runCliScript(__dirname + "/__scripts__/command/command_delete_vsam_data_set.sh",
```

Comment:
The portion of this test that creates the VSAM to delete was removed, and this test does not pass.